### ISSUE TITLE
Level 4 quests cannot be completed

### ISSUE THREAD

**Issue by kneekoo** [cornernote/minetest-skyblock#93]
Digging 10 brown mushrooms gives you an unknown item. This is likely the item required in the last quest, but it's unusable anyway.

![screenshot_20240330_171347](https://github.com/cornernote/minetest-skyblock/assets/566306/8d19dae3-8004-4fd2-8c70-553cca33483a)

As fost the rest of the quests, they can't be completed because of the first one (#91).

**Comment by olliy1or** [comment]
@kneekoo the unknown items are caused by the default farming mod, the farming mod required by this modpack is Farming Redo by TenPlus1 https://content.minetest.net/packages/TenPlus1/farming/

**Comment by olliy1or** [comment]
I've made a PR to fix the issue you had with https://github.com/cornernote/minetest-skyblock/issues/92,

**Comment by kneekoo** [comment]
Nice, thank you! :) Hopefully it gets merged. Does it fix all the issues with level 4?

**Comment by olliy1or** [comment]
> Nice, thank you! :) Hopefully it gets merged. Does it fix all the issues with level 4?

yep, the issue you're having with getting unknown items is because you have the farming mod from mtg, you need to install the mod i linked above.
Also if you're able to test if the PR works, that'd be great. Sometimes "It works on my machine" isn't enough :P

**Comment by SmallJoker** [comment]
Dependency documentation improved in e7a34d0a.
